example_project_app/plugins.py

```diff
@@ -211,6 +216,24 @@ class ProjectAppPlugin(ProjectModifyPluginMixin, ProjectAppPluginPoint):
             'default': '',
             'description': 'Example json project user setting',
         },
+        'project_callable_setting': {
+            'scope': SODAR_CONSTANTS['APP_SETTING_SCOPE_PROJECT'],
+            'type': 'BOOLEAN',
```

Comment:
As mentioned in another comment, I recommend setting the type of these example settings to `STRING`.